### PR TITLE
Johha/refactor return and pick up

### PR DESCRIPTION
**Pull request by johanHardon**
added a check for correct mileage in funtions for both pick up and return. Also added better css for pick up and return. The label for the checkbox "tank full", could be better.

### CODE REVIEW

File: carRental/src/components/CheckBox/CheckBox.jsx [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-      />
-      {label}
-    </label>
+    <label className="check-label">{label}</label>
+    <input
+      type="checkbox"
+      name={label}
+      checked={isSelected}
+      onChange={onCheckboxChange}
+      className="form-check-input"
```
Comment (Evgenios95) [inline carRental/src/components/CheckBox/CheckBox.jsx:11]: Can we change this thingy to checkbox-input instead?

File: carRental/src/components/CheckBox/CheckBox.jsx [hunk 2]
```diff
@@ -1,17 +1,15 @@
 import React from "react";
-
+import "./CheckBox.css";
```
Comment (Evgenios95) [inline carRental/src/components/CheckBox/CheckBox.jsx:2]: Changed all of these class names from form-whatever to checkbox-{whatever}

**Review by Evgenios95 — CHANGES_REQUESTED** [review]

File: carRental/src/components/CheckBox/CheckBox.jsx (hunk 1 above)
Comment (johanHardon) [reply carRental/src/components/CheckBox/CheckBox.jsx]: done

File: carRental/src/components/CheckBox/CheckBox.jsx (hunk 2 above)
Comment (johanHardon) [reply carRental/src/components/CheckBox/CheckBox.jsx:2]: done

**Review by Evgenios95 — APPROVED** [review]

**Pull request merged**